SimPEG/Regularization.py

```diff
@@ -1172,12 +1175,23 @@ def __init__(self, mesh, **kwargs):
         "Model norm scaling to smooth out convergence", default=1.
     )
     epsilon = properties.Float(
-        "Threshold value for the model norm", #, default=1e-1
-        required = True
+        "Threshold value for the model norm", #default=1e-1
+        required=True
     )
     norm = properties.Float(
         "norm used", default=2
     )
+    space = properties.String(
```

Comment:
can we call this something like `coordinate_system`? 
```
coordinate_system = properties.StringChoice(
    "Type of coordinate system we are regularizing in",
    choices=['cartesian', 'spherical'],
    default='cartesian' 
```
http://propertiespy.readthedocs.io/en/latest/content/other.html#stringchoice